internal/pkg/service/buffer/condition/checker.go

```diff
@@ -1,18 +1,25 @@
-package service
+package condition
```

Comment:
Tu je hlavna zmena, `condition.Checker` ma svoj vlastny pkg.

Teda namiesto toho aby bola tato funkcionalita roztrusena po apke, tak staci pozriet do jedneho pkg, kde je to cele implementovane a zdokumetovane.